2023/gaussian-convolution/spanish/sentence_translations.json

```
@@ -2,7 +2,7 @@
  {
   "input": "The basic function underlying a normal distribution, aka a Gaussian, is e to the negative x squared.",
   "model": "nmt",
-  "translatedText": "La función básica subyacente a una distribución normal, también conocida como gaussiana, es e elevado a x negativo al cuadrado.",
+  "translatedText": "La función básica que subyace a una distribución normal, también conocida como gaussiana, es e elevado a menos x cuadrado.",
```

Maybe the way you have it sounds better to other Spanish speakers, but to my north-eastern Colombian ears, "al" makes it more natural:

```suggestion
  "translatedText": "La función básica que subyace a una distribución normal, también conocida como gaussiana, es e elevado a menos x al cuadrado.",
```